tests/tacacs/test_authorization.py

```diff
@@ -417,3 +417,43 @@ def test_backward_compatibility_disable_authorization(
 
     # cleanup
     start_tacacs_server(ptfhost)
+
+
+def test_stop_request_next_server_after_reject(
+        duthosts, enum_rand_one_per_hwsku_hostname,
+        tacacs_creds, ptfhost, check_tacacs, remote_user_client, local_user_client):
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+
+    skip_versions = ["201811", "201911", "202012", "202106", "202111", "202205", "202211"]
```

Comment:
Could you add code comment what version condition will be skipped? I do not get which versions will **not** be skipped.

Reply:
Fixed, the issue fixed by PR https://github.com/sonic-net/sonic-buildimage/pull/14249 is a minor fix. If we cherry-pick the fix to old version, I will update the UT to cover it.